src/file_resolver.rs

```diff
@@ -33,6 +33,18 @@ pub struct ResolvedFile {
     base: PathBuf,
 }
 
+impl ResolvedFile {
+    /// Get the import number associated with this `ResolvedFile`
+    pub fn get_import_no(&self) -> usize {
+        self.import_no
+    }
```

Comment:
Import paths are not always used. For example, if you do `solang compile --target solana examples/solana/flipper.sol` then no import path is used at all. I think `ResolvedFile` `import_no` should be an `Option<usize>`, because it contains garbage in this case (it's set to 0).

Reply:
As the `import_no` and `base` are only used in the solang crate, there is no need for the accessor functions.

Reply:
I added `get_import_no()` because I couldn't access `ResolvedFile.import_no` from `sema::mod::sema_file(...)` (according to my linter, didn't actually try to compile). I added `get_base()` since I was in the area, but am happy to remove it.

Re import paths: I'll update to use `Option<usize>`